frontera/contrib/messagebus/zeromq/__init__.py

```diff
@@ -98,7 +98,7 @@ def flush(self):
         pass
 
     def get_offset(self, partition_id):
-        return self.counters[partition_id]
+        return self.counters.get(partition_id,0)
```

Comment:
Can you please leave a whitespace after the comma in order to pass the failing flake8 tests